### DISCUSSION TITLE
Bot crashes in voice channel

### DISCUSSION THREAD

**Discussion by number0x0001** [discordjs/discord.js#8091] [Q&A] [answered]
For some reason, it's doing this on my linux machine, where I am up today. I get this one windows though. Any help would be appreciated.

![image](https://user-images.githubusercontent.com/73196284/173474156-7a4921f6-055f-4486-b3e8-67f61283f770.png)

**Comment by number0x0001** [comment, marked as answer]
Never mind, I was able to get it figured out. Thanks!

**Reply by Jiralite** [reply]
In future, it would be helpful to know what the error was and what the resolution steps were. Think of someone else with the same issue with no idea how to progress forwards finding your resolved issue with no indication of what to do.
